docs/logagent/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency. 
+We list here just important changes, new features, and critical bug fixes. 
+
+## May 2019 - 2.0.160 - 2.0.170
+
+### New Features
+
+- RedHat container certification
+- Kubernetes audit log plugin to receive data from Kubernetes audit log webhook
+  
+### Improvements
+
+- Docker images: 
+    - Add health check command
+    - Dockerfile for aarch64
+    - Dockerfile for RedHat Enterprise
+- Creation of binary executables using nexe.
+  Binaries can run without installing nodejs runtime. 
+- Automated changelog and release notes
+
+
+### Bugfixes
+
+- Log patterns: Bugfix for web server log patterns
+- Docker plugin: Bugfix for not collecting container logs after quick container restart (timing issue)
```

Comment:
timing => timeout?

Reply:
Actually, it was a timing issue when container did restart within a few seconds. The collector did detach from the container log stream after it was already restarted.